src/common/mi-lttng.c

```diff
@@ -165,10 +164,6 @@ const char * const mi_lttng_loglevel_type_range = "RANGE";
 const char * const mi_lttng_loglevel_type_single = "SINGLE";
 const char * const mi_lttng_loglevel_type_unknown = "UNKNOWN";
 
-/* String related to lttng_calibrate */
-const char * const mi_lttng_element_calibrate = "calibrate";
-const char * const mi_lttng_element_calibrate_function = "FUNCTION";
-
```

Comment:
Distros will flag this as an ABI change. Please move these symbols to src/lib/lttng-ctl/deprecated-symbols.c